worlds/timespinner/Options.py

```diff
@@ -180,12 +180,19 @@ class DamageRandoOverrides(OptionDict):
     }
 
 class HpCap(Range):
-    "Sets the number that Lunais's HP maxes out at."
+    """Sets the number that Lunais's HP maxes out at."""
     display_name = "HP Cap"
     range_start = 1
     range_end = 999
     default = 999
 
+class AuraCap(Range):
+    """Sets the number that Lunais's MP maxes out at."""
```

Comment:
I prefer these descriptions to match the non ap version, so this should be 
`"""Sets the maximum Aura Lunais is allowed to have. Level 1 is 80. Djinn Inferno costs 45"""`